### PR TITLE
fix: Make border around Avatar's badge transparent, not a solid color

### PR DESCRIPTION
**Pull request by behowell**
## Related Issue(s)

- Fixes #

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/77bb9ec1eeedef91315dee3750bd06b15168c693'>77bb9ec1eeedef91315dee3750bd06b15168c693</a></sub>
<div id="bundleSizeComment9423"/>

File: change/@fluentui-react-avatar-e8c58b22-266b-4609-9169-1a26cdef500c.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-avatar-e8c58b22-266b-4609-9169-1a26cdef500c.json:1]: ## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 163 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27292&commitId=42c97fa2102461b226fd0ca57a3b04787ed3fa5a&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1740 screenshots unchanged, 0 screenshots with different dimensions and 163 screenshots with visible difference.


  <details><summary><b>unknown</b> 163 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.basic - Dark Mode.normal.chromium.png|3| Changed|
|Avatar Converged.basic - High Contrast.normal.chromium.png|9| Changed|
|Avatar Converged.basic - RTL.normal.chromium.png|34| Changed|
|Avatar Converged.basic.normal.chromium.png|6| Changed|
|Avatar Converged.customSize+name+badge.normal.chromium.png|83| Changed|
|Avatar Converged.size+icon+badge+square.normal.chromium.png|47| Changed|
|Avatar Converged.size+image+badge.normal.chromium.png|70| Changed|
|Avatar Converged.size+inactive+badge.normal.chromium.png|43| Changed|
|Persona Converged.basic - Dark Mode.normal.chromium.png|1| Changed|
|Persona Converged.basic - High Contrast.normal.chromium.png|5| Changed|
|Persona Converged.basic - RTL.normal.chromium.png|2| Changed|
|Persona Converged.basic.normal.chromium.png|3| Changed|
|Persona Converged.textWrap.normal.chromium.png|3| Changed|
|Table flex - subtle selection.rest.chromium.png|9| Changed|
|Table flex - subtle selection.rest.hover unselected row.chromium.png|9| Changed|
|Table layout flex - cell actions.always visible - Dark Mode.chromium.png|6| Changed|
|Table layout flex - cell actions.always visible - Dark Mode.hover row.chromium.png|7| Changed|
|Table layout flex - cell actions.always visible - High Contrast.chromium.png|19| Changed|
|Table layout flex - cell actions.always visible - High Contrast.hover row.chromium.png|33| Changed|
|Table layout flex - cell actions.always visible - RTL.chromium.png|9| Changed|
|Table layout flex - cell actions.always visible - RTL.hover row.chromium.png|8| Changed|
|Table layout flex - cell actions.always visible.chromium.png|9| Changed|
|Table layout flex - cell actions.always visible.hover row.chromium.png|9| Changed|
|Table layout flex - cell actions.default - Dark Mode.chromium.png|6| Changed|
|Table layout flex - cell actions.default - Dark Mode.hover row.chromium.png|7| Changed|
|Table layout flex - cell actions.default - High Contrast.chromium.png|19| Changed|
|Table layout flex - cell actions.default - High Contrast.hover row.chromium.png|33| Changed|
|Table layout flex - cell actions.default - RTL.chromium.png|9| Changed|
|Table layout flex - cell actions.default - RTL.hover row.chromium.png|8| Changed|
|Table layout flex - cell actions.default.chromium.png|9| Changed|
|Table layout flex - cell actions.default.hover row.chromium.png|9| Changed|
|Table layout flex - cell actions.in header cell.chromium.png|9| Changed|
|Table layout flex - cell actions.in header cell.hover row.chromium.png|9| Changed|
|Table layout flex - headers.sortable - Dark Mode.chromium.png|6| Changed|
|Table layout flex - headers.sortable - Dark Mode.hover header.chromium.png|6| Changed|
|Table layout flex - headers.sortable - Dark Mode.press header.chromium.png|6| Changed|
|Table layout flex - headers.sortable - High Contrast.chromium.png|19| Changed|
|Table layout flex - headers.sortable - High Contrast.hover header.chromium.png|19| Changed|
|Table layout flex - headers.sortable - High Contrast.press header.chromium.png|19| Changed|
|Table layout flex - headers.sortable.chromium.png|9| Changed|
|Table layout flex - headers.sortable.hover header.chromium.png|9| Changed|
|Table layout flex - headers.sortable.press header.chromium.png|9| Changed|
|Table layout flex - truncate.default (disabled).chromium.png|9| Changed|
|Table layout flex - truncate.false.chromium.png|9| Changed|
|Table layout flex - truncate.true.chromium.png|9| Changed|
|Table layout flex.multiselect (checked) - brand - Dark Mode.chromium.png|8| Changed|
|Table layout flex.multiselect (checked) - brand - High Contrast.chromium.png|111| Changed|
|Table layout flex.multiselect (checked) - brand - RTL.chromium.png|7| Changed|
|Table layout flex.multiselect (checked) - brand.chromium.png|11| Changed|
|Table layout flex.multiselect (checked) - neutral - Dark Mode.chromium.png|5| Changed|
|Table layout flex.multiselect (checked) - neutral - High Contrast.chromium.png|76| Changed|
|Table layout flex.multiselect (checked) - neutral - RTL.chromium.png|8| Changed|
|Table layout flex.multiselect (checked) - neutral.chromium.png|13| Changed|
|Table layout flex.multiselect (mixed) - brand - Dark Mode.chromium.png|7| Changed|
|Table layout flex.multiselect (mixed) - brand - High Contrast.chromium.png|65| Changed|
|Table layout flex.multiselect (mixed) - brand - RTL.chromium.png|8| Changed|
|Table layout flex.multiselect (mixed) - brand.chromium.png|10| Changed|
|Table layout flex.multiselect (mixed) - neutral - Dark Mode.chromium.png|6| Changed|
|Table layout flex.multiselect (mixed) - neutral - High Contrast.chromium.png|48| Changed|
|Table layout flex.multiselect (mixed) - neutral - RTL.chromium.png|9| Changed|
|Table layout flex.multiselect (mixed) - neutral.chromium.png|11| Changed|
|Table layout flex.multiselect - Dark Mode.chromium.png|6| Changed|
|Table layout flex.multiselect - High Contrast.chromium.png|19| Changed|
|Table layout flex.multiselect - RTL.chromium.png|9| Changed|
|Table layout flex.multiselect.chromium.png|9| Changed|
|Table layout flex.primary cell.chromium.png|9| Changed|
|Table layout flex.single select (checked) - Dark Mode.chromium.png|6| Changed|
|Table layout flex.single select (checked) - High Contrast.chromium.png|19| Changed|
|Table layout flex.single select (checked) - RTL.chromium.png|9| Changed|
|Table layout flex.single select (checked) - brand - Dark Mode.chromium.png|7| Changed|
|Table layout flex.single select (checked) - brand - High Contrast.chromium.png|42| Changed|
|Table layout flex.single select (checked) - brand - RTL.chromium.png|9| Changed|
|Table layout flex.single select (checked) - brand.chromium.png|10| Changed|
|Table layout flex.single select (checked) - neutral - Dark Mode.chromium.png|6| Changed|
|Table layout flex.single select (checked) - neutral - High Contrast.chromium.png|33| Changed|
|Table layout flex.single select (checked) - neutral - RTL.chromium.png|9| Changed|
|Table layout flex.single select (checked) - neutral.chromium.png|10| Changed|
|Table layout flex.single select (checked).chromium.png|9| Changed|
|Table layout flex.single select - Dark Mode.chromium.png|6| Changed|
|Table layout flex.single select - High Contrast.chromium.png|19| Changed|
|Table layout flex.single select - RTL.chromium.png|9| Changed|
|Table layout flex.single select.chromium.png|9| Changed|
|Table layout flex.size - extra small.chromium.png|7| Changed|
|Table layout flex.size - medium - Dark Mode.chromium.png|6| Changed|
|Table layout flex.size - medium - High Contrast.chromium.png|19| Changed|
|Table layout flex.size - medium - RTL.chromium.png|9| Changed|
|Table layout flex.size - medium.chromium.png|9| Changed|
|Table layout flex.size - small.chromium.png|7| Changed|
|Table layout table - cell actions.always visible - Dark Mode.chromium.png|6| Changed|
|Table layout table - cell actions.always visible - Dark Mode.hover row.chromium.png|7| Changed|
|Table layout table - cell actions.always visible - High Contrast.chromium.png|19| Changed|
|Table layout table - cell actions.always visible - High Contrast.hover row.chromium.png|33| Changed|
|Table layout table - cell actions.always visible - RTL.chromium.png|9| Changed|
|Table layout table - cell actions.always visible - RTL.hover row.chromium.png|8| Changed|
|Table layout table - cell actions.always visible.chromium.png|9| Changed|
|Table layout table - cell actions.always visible.hover row.chromium.png|9| Changed|
|Table layout table - cell actions.default - Dark Mode.chromium.png|6| Changed|
|Table layout table - cell actions.default - Dark Mode.hover row.chromium.png|7| Changed|
|Table layout table - cell actions.default - High Contrast.chromium.png|19| Changed|
|Table layout table - cell actions.default - High Contrast.hover row.chromium.png|33| Changed|
|Table layout table - cell actions.default - RTL.chromium.png|9| Changed|
|Table layout table - cell actions.default - RTL.hover row.chromium.png|8| Changed|
|Table layout table - cell actions.default.chromium.png|9| Changed|
|Table layout table - cell actions.default.hover row.chromium.png|9| Changed|
|Table layout table - cell actions.in header cell.chromium.png|9| Changed|
|Table layout table - cell actions.in header cell.hover row.chromium.png|9| Changed|
|Table layout table - headers.sortable - Dark Mode.chromium.png|6| Changed|
|Table layout table - headers.sortable - Dark Mode.hover header.chromium.png|6| Changed|
|Table layout table - headers.sortable - Dark Mode.press header.chromium.png|6| Changed|
|Table layout table - headers.sortable - High Contrast.chromium.png|19| Changed|
|Table layout table - headers.sortable - High Contrast.hover header.chromium.png|19| Changed|
|Table layout table - headers.sortable - High Contrast.press header.chromium.png|19| Changed|
|Table layout table - headers.sortable.chromium.png|9| Changed|
|Table layout table - headers.sortable.hover header.chromium.png|9| Changed|
|Table layout table - headers.sortable.press header.chromium.png|9| Changed|
|Table layout table - truncate.default (disabled).chromium.png|9| Changed|
|Table layout table - truncate.false.chromium.png|9| Changed|
|Table layout table - truncate.true.chromium.png|9| Changed|
|Table layout table.multiselect (checked) - brand - Dark Mode.chromium.png|8| Changed|
|Table layout table.multiselect (checked) - brand - High Contrast.chromium.png|111| Changed|
|Table layout table.multiselect (checked) - brand - RTL.chromium.png|7| Changed|
|Table layout table.multiselect (checked) - brand.chromium.png|11| Changed|
|Table layout table.multiselect (checked) - neutral - Dark Mode.chromium.png|5| Changed|
|Table layout table.multiselect (checked) - neutral - High Contrast.chromium.png|76| Changed|
|Table layout table.multiselect (checked) - neutral - RTL.chromium.png|8| Changed|
|Table layout table.multiselect (checked) - neutral.chromium.png|13| Changed|
|Table layout table.multiselect (mixed) - brand - Dark Mode.chromium.png|7| Changed|
|Table layout table.multiselect (mixed) - brand - High Contrast.chromium.png|65| Changed|
|Table layout table.multiselect (mixed) - brand - RTL.chromium.png|8| Changed|
|Table layout table.multiselect (mixed) - brand.chromium.png|10| Changed|
|Table layout table.multiselect (mixed) - neutral - Dark Mode.chromium.png|6| Changed|
|Table layout table.multiselect (mixed) - neutral - High Contrast.chromium.png|48| Changed|
|Table layout table.multiselect (mixed) - neutral - RTL.chromium.png|9| Changed|
|Table layout table.multiselect (mixed) - neutral.chromium.png|11| Changed|
|Table layout table.multiselect - Dark Mode.chromium.png|6| Changed|
|Table layout table.multiselect - High Contrast.chromium.png|19| Changed|
|Table layout table.multiselect - RTL.chromium.png|9| Changed|
|Table layout table.multiselect.chromium.png|9| Changed|
|Table layout table.primary cell.chromium.png|9| Changed|
|Table layout table.single select (checked) - Dark Mode.chromium.png|6| Changed|
|Table layout table.single select (checked) - High Contrast.chromium.png|19| Changed|
|Table layout table.single select (checked) - RTL.chromium.png|9| Changed|
|Table layout table.single select (checked) - brand - Dark Mode.chromium.png|7| Changed|
|Table layout table.single select (checked) - brand - High Contrast.chromium.png|42| Changed|
|Table layout table.single select (checked) - brand - RTL.chromium.png|9| Changed|
|Table layout table.single select (checked) - brand.chromium.png|10| Changed|
|Table layout table.single select (checked) - neutral - Dark Mode.chromium.png|6| Changed|
|Table layout table.single select (checked) - neutral - High Contrast.chromium.png|33| Changed|
|Table layout table.single select (checked) - neutral - RTL.chromium.png|9| Changed|
|Table layout table.single select (checked) - neutral.chromium.png|10| Changed|
|Table layout table.single select (checked).chromium.png|9| Changed|
|Table layout table.single select - Dark Mode.chromium.png|6| Changed|
|Table layout table.single select - High Contrast.chromium.png|19| Changed|
|Table layout table.single select - RTL.chromium.png|9| Changed|
|Table layout table.single select.chromium.png|9| Changed|
|Table layout table.size - extra small.chromium.png|7| Changed|
|Table layout table.size - medium - Dark Mode.chromium.png|6| Changed|
|Table layout table.size - medium - High Contrast.chromium.png|19| Changed|
|Table layout table.size - medium - RTL.chromium.png|9| Changed|
|Table layout table.size - medium.chromium.png|9| Changed|
|Table layout table.size - small.chromium.png|7| Changed|
|Table table - subtle selection.rest.chromium.png|9| Changed|
|Table table - subtle selection.rest.hover unselected row.chromium.png|9| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27292/perf-test-react-components/Avatar-mount_base.html">1078</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27292/perf-test-react-components/Avatar-mount.html">1108</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27292/perf-test-react-components/Button-mount_base.html">694</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27292/perf-test-react-components/Button-mount.html">680</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27292/perf-test-react-components/Field-mount_base.html">1956</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27292/perf-test-react-components/Field-mount.html">1948</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27292/perf-test-react-components/FluentProvider-mount_base.html">1371</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27292/perf-test-react-components/FluentProvider-mount.html">1416</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27292/perf-test-react-components/FluentProviderWithTheme-mount_base.html">383</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27292/perf-test-react-components/FluentProviderWithTheme-mount.html">381</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27292/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">290</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27292/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">278</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27292/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">356</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27292/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">342</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27292/perf-test-react-components/InfoButton-mount_base.html">249</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27292/perf-test-react-components/InfoButton-mount.html">257</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27292/perf-test-react-components/MakeStyles-mount_base.html">1552</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27292/perf-test-react-components/MakeStyles-mount.html">1576</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27292/perf-test-react-components/Persona-mount_base.html">2664</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27292/perf-test-react-components/Persona-mount.html">2708</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27292/perf-test-react-components/SpinButton-mount_base.html">2326</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27292/perf-test-react-components/SpinButton-mount.html">2397</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by behowell** [comment]
Abandoning in favor of #27527

**Pull request closed without merging**